requirements.txt

```diff
@@ -1,7 +1,11 @@
+bzr==2.8.0.dev1
```

Comment:
Some of these requirements appear to be unrelated to the Flask or psutil install. If you install both of those in a fresh venv you should get the current list of up-to-date requirements needed.